### ISSUE TITLE
Upload our app icon to the app

### ISSUE THREAD

**Issue by BurningFenix** [RUMobileTeam/RUMobile#4]
Find a way to upload icon to our react native project
<img width="180" alt="screen shot 2016-08-03 at 4 47 35 pm" src="https://user-images.githubusercontent.com/4228915/27343825-27161c9c-55b2-11e7-9ed8-72ecaf8293b9.png">

**Comment by ropc** [comment]
Pretty sure you would do this in the respective `ios/` and `android/` project files. could be wrong though

**Comment by BurningFenix** [comment]
Yeah I'm opening a issue for every task we must do. Some of them might seem obvious but I'll still open them until we figure it out :P
